bin/regexp-tree

```diff
@@ -3,53 +3,44 @@
 'use strict';
 
 const colors = require('colors');
-const fs = require('fs');
 const os = require('os');
 const regexpTree = require('..');
 
-const options = require('nomnom')
-  .script('regexp-tree')
+const options = require('yargs')
+  .usage('Usage: $0 [options]')
   .options({
     expression: {
-      abbr: 'e',
-      help: 'A regular expression to be parsed',
-      type: 'string',
+      alias: 'e',
+      describe: 'A regular expression to be parsed',
       required: true,
+      nargs: 1,
```

Comment:
`required` has been deprecated in favor of `demandOption`.

Also, I find it quite unbelievable that yargs does not provide a way to prevent an arg from being set twice... (Yet see yargs/yargs#465)

Currently, running `regexp-tree -e "/foo/" -e "/bar/"` leads to `e` being an array containing both strings... This behaviour can be disabled globally but I don't think we want to do this since repeated arguments might come in handy in the future.

Maybe we could add an additional coerce option:
`coerce: v => Array.isArray(v) ? v[v.length - 1] : v`
to keep only the last value if the arg's been repeated.

Reply:
You can use `requiresArg: true` instead of `nargs: 1`, although I'm not exactly sure what the difference is.

Reply:
OK, sounds good; yeah, I'd expect `enforceOne` or `unique`, or something instead of `coerce`, but I'll add one.